parts/k8s/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -9,11 +9,12 @@ if ! echo "${UBUNTU_OS_NAME} ${RHEL_OS_NAME} ${DEBIAN_OS_NAME}" | grep -q "${OS}
 fi
 KUBECTL=/usr/local/bin/kubectl
 DOCKER=/usr/bin/docker
-GPU_DV=418.40.04
-GPU_DEST=/usr/local/nvidia
+export GPU_DV=418.40.04
```

Comment:
These vars are used by the exported `installNvidiaDrivers` func, so these export statements make them available to that forked execution context.